src/plugins/auth.js

```diff
@@ -36,20 +36,29 @@ export const auth = {
          */
         validate: (artifacts) => {
           const user = artifacts.decoded.payload
-          const groups = Array.isArray(user?.groups) ? user.groups : []
 
-          logger.debug(`Validating user against groups: ${groups.join(', ')}`)
+          if (!user) {
+            return {
+              isValid: false
+            }
+          }
+
+          const groups = Array.isArray(user.groups) ? user.groups : []
```

Comment:
Is groups sometimes not an array? Default is good

```suggestion
          const { groups = [] } = user
```

Reply:
Enhancement in PR https://github.com/DEFRA/forms-manager/pull/176.

Commit https://github.com/DEFRA/forms-manager/pull/176/commits/45a41ff4cf9f7b2c8bfcf5de352d031b1cf54c28.